Add `ResizeObserver` to `ViewportComponent` as a workaround for parent zero dimension uncaught exception (#1416)
iTwin/appui: .changeset/funny-ads-write.md

---
"@itwin/imodel-components-react": patch
---

> add a resize observer to the ViewportComponent as a workaround to throwing error when parent div has a zero dimension

Reviewer: ```suggestion
Fixed zero dimension error in `ViewportComponent` by using a resize observer.
```
